src/lightning_app/cli/lightning_cli.py

```diff
@@ -206,15 +290,63 @@ def stop():
     pass
 
 
-@_main.group(hidden=True)
+@_main.group(name="delete")
 def delete():
-    """Delete an application."""
+    """Delete a Lightning managed resources."""
     pass
 
 
-@_main.group(name="list", hidden=True)
+@delete.command("cluster")
+@click.argument("cluster", type=str)
+@click.option(
+    "--force",
+    "force",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="""Delete a cluster from Lightning AI. This does NOT delete any resources created by the cluster,
```

Comment:
Please change to:

Delete a BYOC cluster from Lightning AI. This does NOT delete any resources created by the cluster

Reply:
:+1: